### PR TITLE
trace: add trace quickstart

### PR DESCRIPTION
**Pull request by rakyll**

### CODE REVIEW

File: trace/trace_quickstart/main.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
+// Copyright 2017 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// [START trace_quickstart]
```
Comment (broady) [inline trace/trace_quickstart/main.go]: add a newline so it doesn't show in package doc

File: trace/trace_quickstart/main.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	// Sets your Google Cloud Platform project ID.
+	projectID := "YOUR_PROJECT_ID"
+
+	// Creates a client.
+	traceClient, err := trace.NewClient(ctx, projectID)
+	if err != nil {
+		log.Fatalf("Failed to create client: %v", err)
+	}
+
+	httpClient := http.Client{
```
Comment (broady) [inline trace/trace_quickstart/main.go]: &http.Client

File: trace/trace_quickstart/main.go [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+		Transport: &trace.Transport{},
+	}
+
+	handler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		req, _ := http.NewRequest("GET", "https://metadata/users", nil)
+		req = req.WithContext(r.Context())
```
Comment (broady) [inline trace/trace_quickstart/main.go]: A comment for this line, perhaps? This is the part that propagates the Trace ID, right?

File: trace/trace_quickstart/main.go (hunk 1 above)
Comment (rakyll) [inline trace/trace_quickstart/main.go]: Done

File: trace/trace_quickstart/main.go (hunk 2 above)
Comment (rakyll) [inline trace/trace_quickstart/main.go]: Done

File: trace/trace_quickstart/main.go (hunk 3 above)
Comment (rakyll) [inline trace/trace_quickstart/main.go]: Done

**Comment by rakyll** [comment]
All addressed.

**Comment by 00bentleyg** [comment]
bb

**Comment by 00bentleyg** [comment]
rakyll you say author many maintainer of none. you are like intern. nothing to brag about. you are not belong in tech industry woman

**Pull request merged**